tools/apidiff/report/packages.go

```diff
@@ -159,17 +173,25 @@ func (r PkgsReport) IsEmpty() bool {
 }
 
 // ToMarkdown writes the report to string in the markdown form
-func (r *PkgsReport) ToMarkdown() string {
+func (r *PkgsReport) ToMarkdown(withHeader bool, version, marker string) string {
```

Comment:
Why do we need the `withHeader` param?

Reply:
I added this so that anybody creating a new command and who only wants the pure diff without the headers can do so by calling this with the bool set to false.

Reply:
I'll also add a comment about this to explain that

Reply:
I see.  Looking at the code, it would appear that the `version` param is only used when `withHeader` is `true`.  Perhaps we can condense this to just `withHeader`, and if it's not the empty string write the header with the version.  What do you think?

Reply:
Yeah, that's a good idea. I'll change it.

Reply:
Done